src/assets.ts

```diff
@@ -298,9 +363,14 @@ export function addAssetsIfNecessary(server: OmnisharpServer) {
         return;
     }
     
-    // If there is no project.json, we won't bother to prompt the user for tasks.json.		
-    const projectJsonPath = path.join(vscode.workspace.rootPath, 'project.json');		
-    if (!fs.existsSync(projectJsonPath)) {
+    // If there is no project.json or global.json, we won't bother to prompt the user for tasks.json.		
```

Comment:
>global.json [](start = 38, length = 11)

Now that we know how to build a project lower down, do we need this check? Ex: if I have:

src
   MyExe
   MyLib

And I open the src folder, shouldn't we handle this case?